src/components/Sheet/usePlanSheetFill.tsx

```diff
@@ -519,10 +519,12 @@ export const usePlanSheetFill = ({
           psgType={rowInfo[rowIndex].passageType}
           published={rowInfo[rowIndex].published}
           organizedBy={organizedBy}
-          sectionSequenceNumber={row[SectionSeqCol].toString()}
-          passageSequenceNumber={row[
-            passageSeqCol >= 0 ? passageSeqCol : 0
-          ].toString()}
+          sectionSequenceNumber={positiveWholeOnly(
+            row[SectionSeqCol] as number
+          )}
+          passageSequenceNumber={positiveWholeOnly(
+            row[passageSeqCol >= 0 ? passageSeqCol : 0] as number
```

Comment:
do you still need to do this check out here also...won't positiveWholeOnly figure it out?

Reply:
This test is checking to see if there is a passage sequence number column. In a flat project, there isn't one.